test/e2e/tests/metamask-responsive-ui.spec.js

```diff
@@ -0,0 +1,233 @@
+        await driver.clickElement({ text: 'Create a Wallet', tag: 'button' });
+
+        // clicks the "I Agree" option on the metametrics opt-in screen
+        await driver.clickElement('.btn-primary');
+
+        // accepts a secure password
+        await driver.fill(
+          '.first-time-flow__form #create-password',
+          'correct horse battery staple',
+        );
+        await driver.fill(
+          '.first-time-flow__form #confirm-password',
+          'correct horse battery staple',
+        );
+        await driver.clickElement('.first-time-flow__checkbox');
+        await driver.clickElement('.first-time-flow__form button');
+
+        // reveals the seed phrase
+        await driver.clickElement(
+          '.reveal-seed-phrase__secret-blocker .reveal-seed-phrase__reveal-button',
+        );
+        const revealedSeedPhrase = await driver.findElement(
+          '.reveal-seed-phrase__secret-words',
+        );
+        const seedPhrase = await revealedSeedPhrase.getText();
+        assert.equal(seedPhrase.split(' ').length, 12);
+
+        await driver.clickElement({
+          text: enLocaleMessages.next.message,
+          tag: 'button',
+        });
+
+        async function clickWordAndWait(word) {
+          await driver.clickElement(
+            `[data-testid="seed-phrase-sorted"] [data-testid="draggable-seed-${word}"]`,
+          );
+          await driver.delay(tinyDelayMs);
+        }
+
+        // can retype the seed phrase
+        const words = seedPhrase.split(' ');
+        for (const word of words) {
+          await clickWordAndWait(word);
+        }
+        await driver.clickElement({ text: 'Confirm', tag: 'button' });
+
+        // clicks through the success screen
+        await driver.findElement({ text: 'Congratulations', tag: 'div' });
+        await driver.clickElement({
+          text: enLocaleMessages.endOfFlowMessage10.message,
+          tag: 'button',
+        });
+
+        // Show account information
+        // show account details dropdown menu
+        await driver.clickElement(
+          '[data-testid="account-options-menu-button"]',
+        );
+        const options = await driver.findElements(
+          '.account-options-menu .menu-item',
+        );
+        assert.equal(options.length, 3); // HD Wallet type does not have to show the Remove Account option
+        // click outside of menu to dismiss
+        // account menu button chosen because the menu never covers it.
+        await driver.clickPoint('.account-menu__icon', 0, 0);
+
+        // logs out of the vault
+        await driver.clickElement('.account-menu__icon');
+
+        const lockButton = await driver.findClickableElement(
+          '.account-menu__lock-button',
+        );
+        assert.equal(await lockButton.getText(), 'Lock');
+        await lockButton.click();
+      },
+    );
+  });
+
+  it('Importing existing wallet from login page', async function () {
```

Comment:
Nit: we've been calling this the "lock" page (unlock/lock), instead of the "login" page

```suggestion
  it('Importing existing wallet from lock page', async function () {
```